modules/virt-hot-unplugging-a-virtual-disk.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-unplugging-a-virtual-disk_{context}"]
+= Hot-unplugging a virtual disk
+
+Hot-unplug virtual disks that you want to detach from a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* Your virtual machine must be running to hot-unplug the virtual machine's virtual disk.
+* You must have at least one data volume or persistent volume claim (PVC) available and hot-plugged, using the `addvolume` command with `virtctl`.
+
+To hot-unplug a virtual disk using the CLI, use the `removevolume` command with `virtctl`.
```

Comment:
See comment above about lead-in sentences

Reply:
Done.